### ISSUE TITLE
[BUG] HugeCTR backend   crashed when I make pressure test with 100 concurrents，and the coredump file not generated.

### ISSUE THREAD

**Issue by offbye** [triton-inference-server/hugectr_backend#27]
HugeCTR backend   crashed when I make pressure test with 100 concurrents . and the coredump file not generated.

```
I1215 03:27:54.113361 33740 hugectr.cc:1438] model h3_hugectr, instance h3_hugectr, executing 1 requests
I1215 03:27:54.113399 33740 hugectr.cc:1512] request 0: id = "1", correlation_id = 0, input_count = 3, requested_output_count = 1
I1215 03:27:54.113373 33740 http_server.cc:2727] HTTP request: 2 /v2/models/h3_hugectr/infer
I1215 03:27:54.113420 33740 hugectr.cc:1610]    input CATCOLUMN: datatype = INT64, shape = [1,576], byte_size = 4608, buffer_count = 1
I1215 03:27:54.113434 33740 model_repository_manager.cc:615] GetInferenceBackend() 'h3_hugectr' version -1
I1215 03:27:54.113466 33740 model_repository_manager.cc:615] GetInferenceBackend() 'h3_hugectr' version -1
I1215 03:27:54.113455 33740 hugectr.cc:1626]    input ROWINDEX: datatype = INT32, shape = [1,577], byte_size = 2308, buffer_count = 1
I1215 03:27:54.113478 33740 hugectr.cc:1640]    input DES: datatype = FP32, shape = [1,704], byte_size = 2816, buffer_count = 1
I1215 03:27:54.113484 33740 hugectr.cc:1665]    requested_output OUTPUT0
I1215 03:27:54.113495 33740 infer_response.cc:165] add response output: output: OUTPUT0, type: FP32, shape: [32]
I1215 03:27:54.113493 33740 infer_request.cc:524] prepared: [0x0x7f1f0800d230] request id: 1, model: h3_hugectr, requested version: -1, actual version: 1, flags: 0x0, correlation id: 0, batch size: 1, priority: 0, timeout (us): 0
original inputs:
[0x0x7f1f0800aa88] input: ROWINDEX, type: INT32, original shape: [1,577], batch + shape: [1,577], shape: [577]
[0x0x7f1f0800edc8] input: CATCOLUMN, type: INT64, original shape: [1,576], batch + shape: [1,576], shape: [576]
[0x0x7f1f0800b7d8] input: DES, type: FP32, original shape: [1,704], batch + shape: [1,704], shape: [704]
override inputs:
inputs:
[0x0x7f1f0800b7d8] input: DES, type: FP32, original shape: [1,704], batch + shape: [1,704], shape: [704]
[0x0x7f1f0800edc8] input: CATCOLUMN, type: INT64, original shape: [1,576], batch + shape: [1,576], shape: [576]
[0x0x7f1f0800aa88] input: ROWINDEX, type: INT32, original shape: [1,577], batch + shape: [1,577], shape: [577]
original requested outputs:
OUTPUT0
requested outputs:
OUTPUT0

I1215 03:27:54.113504 33740 http_server.cc:1051] HTTP: unable to provide 'OUTPUT0' in GPU, will use CPU
I1215 03:27:54.113543 33740 http_server.cc:1071] HTTP using buffer for: 'OUTPUT0', size: 128, addr: 0x7f1f44bca620
I1215 03:27:54.113581 33740 hugectr.cc:1438] model h3_hugectr, instance h3_hugectr, executing 1 requests
I1215 03:27:54.113599 33740 hugectr.cc:1512] request 0: id = "1", correlation_id = 0, input_count = 3, requested_output_count = 1
I1215 03:27:54.113615 33740 hugectr.cc:1610]    input CATCOLUMN: datatype = INT64, shape = [1,576], byte_size = 4608, buffer_count = 1
I1215 03:27:54.113629 33740 hugectr.cc:1626]    input ROWINDEX: datatype = INT32, shape = [1,577], byte_size = 2308, buffer_count = 1
I1215 03:27:54.113640 33740 hugectr.cc:1640]    input DES: datatype = FP32, shape = [1,704], byte_size = 2816, buffer_count = 1
I1215 03:27:54.113646 33740 hugectr.cc:1665]    requested_output OUTPUT0
I1215 03:27:54.113651 33740 infer_response.cc:165] add response output: output: OUTPUT0, type: FP32, shape: [32]
I1215 03:27:54.113663 33740 http_server.cc:1051] HTTP: unable to provide 'OUTPUT0' in GPU, will use CPU
I1215 03:27:54.113674 33740 http_server.cc:1071] HTTP using buffer for: 'OUTPUT0', size: 128, addr: 0x7f1fdcbc3b60
I1215 03:27:54.115386 33740 hugectr.cc:1815] *****Processing request on device***** 0 for model h3_hugectr
I1215 03:27:54.120122 33740 hugectr.cc:1815] *****Processing request on device***** 0 for model h3_hugectr
terminate called after throwing an instance of 'std::system_error'
  what():  Resource temporarily unavailable
Signal (6) received.
terminate called recursively
Signal (6) received.
 0# 0x0000555E676798A9 in tritonserver
 1# 0x00007F22576FF210 in /usr/lib/x86_64-linux-gnu/libc.so.6
 2# gsignal in /usr/lib/x86_64-linux-gnu/libc.so.6
 3# abort in /usr/lib/x86_64-linux-gnu/libc.so.6
 4# 0x00007F2257AB5911 in /usr/lib/x86_64-linux-gnu/libstdc++.so.6
 5# 0x00007F2257AC138C in /usr/lib/x86_64-linux-gnu/libstdc++.so.6
 6# 0x00007F2257AC13F7 in /usr/lib/x86_64-linux-gnu/libstdc++.so.6
 7# 0x00007F2257AC16A9 in /usr/lib/x86_64-linux-gnu/libstdc++.so.6
 8# std::__throw_system_error(int) in /usr/lib/x86_64-linux-gnu/libstdc++.so.6
 9# 0x00007F2257AEE0BD in /usr/lib/x86_64-linux-gnu/libstdc++.so.6
10# HugeCTR::embedding_cache<long long>::look_up(void const*, std::vector<unsigned long, std::allocator<unsigned long> > const&, float*, HugeCTR::MemoryBlock*, std::vector<CUstream_st*, std::allocator<CUstream_st*> > const&, float) in /usr/local/hugectr/lib/libhugectr_inference.so
11# HugeCTR::InferenceSession::predict(float*, void*, int*, float*, int) in /usr/local/hugectr/lib/libhugectr_inference.so
12# 0x00007F22383F404A in /opt/tritonserver/backends/hugectr/libtriton_hugectr.so
13# TRITONBACKEND_ModelInstanceExecute in /opt/tritonserver/backends/hugectr/libtriton_hugectr.so
14# 0x00007F225828B83A in /opt/tritonserver/bin/../lib/libtritonserver.so
15# 0x00007F225828C04D in /opt/tritonserver/bin/../lib/libtritonserver.so
16# 0x00007F2258140801 in /opt/tritonserver/bin/../lib/libtritonserver.so
17# 0x00007F2258285DC7 in /opt/tritonserver/bin/../lib/libtritonserver.so
18# 0x00007F2257AEDDE4 in /usr/lib/x86_64-linux-gnu/libstdc++.so.6
19# 0x00007F2257F6B609 in /usr/lib/x86_64-linux-gnu/libpthread.so.0
20# clone in /usr/lib/x86_64-linux-gnu/libc.so.6

Aborted (core dumped)
root@13609cdec7a0:/opt/tritonserver# ll
total 3612
drwxr-xr-x 1 root          root             4096 Nov 29 08:07  ./
drwxr-xr-x 1 root          root             4096 Oct 21 18:44  ../
-rw-r--r-- 1 root          root              641 Nov  4 19:20 '=1.21.6'
-rw-rw-r-- 1 triton-server triton-server    1485 Oct 21 17:20  LICENSE
-rw-rw-r-- 1 triton-server triton-server 3012640 Oct 21 17:20  NVIDIA_Deep_Learning_Container_License.pdf
-rw-rw-r-- 1 triton-server triton-server       7 Oct 21 17:20  TRITON_VERSION
drwxr-xr-x 1 triton-server triton-server    4096 Nov  4 19:33  backends/
drwxr-xr-x 2 triton-server triton-server    4096 Oct 21 18:45  bin/
drwxr-xr-x 1 triton-server triton-server    4096 Oct 21 18:45  include/
drwxr-xr-x 2 triton-server triton-server    4096 Oct 21 18:45  lib/
-rw------- 1 root          root           616982 Dec 10 02:29  nohup.out
-rwxrwxr-x 1 triton-server triton-server    4090 Oct 21 17:20  nvidia_entrypoint.sh*
drwxr-xr-x 3 triton-server triton-server    4096 Oct 21 18:48  repoagents/
drwxr-xr-x 2 triton-server triton-server    4096 Oct 21 18:45  third-party-src/
root@13609cdec7a0:/opt/tritonserver# ll /opt/tritonserver/backends/hugectr/
total 400
drwxr-xr-x 2 root root   4096 Nov  4 19:33 ./
drwxr-xr-x 3 root root   4096 Nov  4 19:33 ../
-rw-r--r-- 1 root root 398904 Nov  4 19:33 libtriton_hugectr.so


```

## my start command：
```
 tritonserver --model-repository=/data/gux/h3/h3_hugectr/ --backend-config=hugectr,ps=/data/gux/h3/h3_hugectr/ps.json --load-model h3_hugectr --model-control-mode=explicit --log-verbose 1 

#pressure test  command by siege

$ siege -H "Content-Type:application/json" "http://127.0.0.1:8088/xxxx/v1/api  POST < /home/t3mgr/data.json" -b -c 100  -t 5M

```
I1215 03:44:22.466890 66071 hugectr.cc:748] The model configuration:
{
    "name": "h3_hugectr",
    "platform": "",
    "backend": "hugectr",
    "version_policy": {
        "latest": {
            "num_versions": 1
        }
    },
    "max_batch_size": 4096,
    "input": [
        {
            "name": "DES",
            "data_type": "TYPE_FP32",
            "format": "FORMAT_NONE",
            "dims": [
                -1
            ],
            "is_shape_tensor": false,
            "allow_ragged_batch": false
        },
        {
            "name": "CATCOLUMN",
            "data_type": "TYPE_INT64",
            "format": "FORMAT_NONE",
            "dims": [
                -1
            ],
            "is_shape_tensor": false,
            "allow_ragged_batch": false
        },
        {
            "name": "ROWINDEX",
            "data_type": "TYPE_INT32",
            "format": "FORMAT_NONE",
            "dims": [
                -1
            ],
            "is_shape_tensor": false,
            "allow_ragged_batch": false
        }
    ],
    "output": [
        {
            "name": "OUTPUT0",
            "data_type": "TYPE_FP32",
            "dims": [
                -1
            ],
            "label_filename": "",
            "is_shape_tensor": false
        }
    ],
    "batch_input": [],
    "batch_output": [],
    "optimization": {
        "priority": "PRIORITY_DEFAULT",
        "input_pinned_memory": {
            "enable": true
        },
        "output_pinned_memory": {
            "enable": true
        },
        "gather_kernel_buffer_threshold": 0,
        "eager_batching": false
    },
    "instance_group": [
        {
            "name": "h3_hugectr_0",
            "kind": "KIND_GPU",
            "count": 3,
            "gpus": [
                0
            ],
            "secondary_devices": [],
            "profile": [],
            "passive": false,
            "host_policy": ""
        }
    ],
    "default_model_filename": "",
    "cc_model_filenames": {},
    "metric_tags": {},
    "parameters": {
        "cat_feature_num": {
            "string_value": "18"
        },
        "config": {
            "string_value": "/data/gux/h3/1/dcn.json"
        },
        "label_dim": {
            "string_value": "1"
        },
        "max_nnz": {
            "string_value": "1"
        },
        "embedding_vector_size": {
            "string_value": "128"
        },
        "gpucacheper": {
            "string_value": "0.5"
        },
        "des_feature_num": {
            "string_value": "22"
        },
        "gpucache": {
            "string_value": "true"
        },
        "embeddingkey_long_type": {
            "string_value": "true"
        },
        "slots": {
            "string_value": "18"
        }
    },
    "model_warmup": []
}

**Comment by yingcanw** [comment]
Since the issue you encountered cannot be reproduced, we recommend that you use the [performance testing tool](https://github.com/triton-inference-server/server/blob/main/docs/perf_analyzer.md#request-concurrency) provided by Triton , which includes a concurrent testing mechanism. 
We also provide use cases in the [dlrm inference notebook](https://github.com/triton-inference-server/hugectr_backend/blob/main/samples/dlrm/HugeCTR_DLRM_Inference.ipynb)(**5. Get Inference benchmark by Triton Performance Tool**)for your reference .

**Comment by offbye** [comment]
use docker container: 
13609cdec7a0   nvcr.io/nvidia/merlin/merlin-inference:21.11   "/bin/bash"   3 weeks ago   Up 3 weeks   0.0.0.0:8100-8102->8100-8102/tcp, :::8100-8102->8100-8102/tcp                                            


按照/hugectr_backend/samples/dlrm/HugeCTR_DLRM_Inference.ipynb 使用perf_analyzer压测， triton crashed


压测命令
perf_analyzer -m dlrm -u localhost:8100 --input-data 1.json --shape CATCOLUMN:26 --shape DES:13 --shape ROWINDEX:27   --concurrency-range 1:4 


Successfully read data for 1 stream/streams with 1 step/steps.
*** Measurement Settings ***
  Batch size: 1
  Using "time_windows" mode for stabilization
  Measurement window: 5000 msec
  Latency limit: 0 msec
  Concurrency limit: 4 concurrent requests
  Using synchronous calls for inference
  Stabilizing using average latency

Request concurrency: 1
  Client:
    Request count: 6378
    Throughput: 1275.6 infer/sec
    Avg latency: 777 usec (standard deviation 133 usec)
    p50 latency: 761 usec
    p90 latency: 836 usec
    p95 latency: 885 usec
    p99 latency: 1105 usec
    Avg HTTP time: 771 usec (send/recv 36 usec + response wait 735 usec)
  Server:
    Inference count: 7664
    Execution count: 7664
    Successful request count: 7664
    Avg request latency: 612 usec (overhead 1 usec + queue 136 usec + compute input 0 usec + compute infer 475 usec + compute output 0 usec)

Request concurrency: 2
Failed to update context stat: Failed to update context stat: Timer not set correctly.
Timer not set correctly.
Failed to maintain requested inference load. Worker thread(s) failed to generate concurrent requests.
Thread [0] had error: failed to parse the request JSON buffer: The document is empty. at 0
Thread [1] had error: failed to parse the request JSON buffer: The document is empty. at 0


启动模型服务
root@13609cdec7a0:/opt/tritonserver# tritonserver --model-repository=/data/dlrm_train/dlrm_infer/model/ --load-model=dlrm --backend-config=hugectr,ps=/data/dlrm_train/dlrm_infer/model/ps.json  --backend-directory=/usr/local/hugectr/backends  --model-control-mode=explicit    --grpc-port 8101  --http-port 8100    --metrics-port 8102



错误日志

[HUGECTR][02:16:08][INFO][RANK0]: Parameter server lookup of 0 / 0 embeddings took 155 us.
I1220 02:16:08.129594 539261 hugectr.cc:1832] Prediction execution time is 0 ms
I1220 02:16:08.129628 539261 hugectr.cc:1438] model dlrm, instance dlrm, executing 1 requests
I1220 02:16:08.129638 539261 hugectr.cc:1512] request 0: id = "", correlation_id = 0, input_count = 3, requested_output_count = 1
I1220 02:16:08.129646 539261 hugectr.cc:1610]   input CATCOLUMN: datatype = INT64, shape = [1,26], byte_size = 208, buffer_count = 1
I1220 02:16:08.129652 539261 hugectr.cc:1626]   input ROWINDEX: datatype = INT32, shape = [1,27], byte_size = 108, buffer_count = 1
I1220 02:16:08.129663 539261 hugectr.cc:1640]   input DES: datatype = FP32, shape = [1,13], byte_size = 52, buffer_count = 1
I1220 02:16:08.129672 539261 hugectr.cc:1665]   requested_output OUTPUT0
I1220 02:16:08.129713 539261 hugectr.cc:1815] *****Processing request on device***** 0 for model dlrm
[HUGECTR][02:16:08][INFO][RANK0]: Looking up 0 embeddings (each with 128 values)...
[HUGECTR][02:16:08][INFO][RANK0]: ParallelLocalMemory backend. Table: dlrm#0. Fetched 0 / 0 values.
[HUGECTR][02:16:08][INFO][RANK0]: ParallelLocalMemory: 0 hits, 0 missing!
[HUGECTR][02:16:08][INFO][RANK0]: Parameter server lookup of 0 / 0 embeddings took 139 us.
I1220 02:16:08.130077 539261 hugectr.cc:1823] ******Processing request completed!*****
I1220 02:16:08.130116 539261 hugectr.cc:1832] Prediction execution time is 0 ms
I1220 02:16:08.130153 539261 hugectr.cc:1438] model dlrm, instance dlrm, executing 1 requests
I1220 02:16:08.130164 539261 hugectr.cc:1512] request 0: id = "", correlation_id = 0, input_count = 3, requested_output_count = 1
I1220 02:16:08.130177 539261 hugectr.cc:1610]   input CATCOLUMN: datatype = INT64, shape = [1,26], byte_size = 208, buffer_count = 1
I1220 02:16:08.130192 539261 hugectr.cc:1626]   input ROWINDEX: datatype = INT32, shape = [1,27], byte_size = 108, buffer_count = 1
I1220 02:16:08.130201 539261 hugectr.cc:1640]   input DES: datatype = FP32, shape = [1,13], byte_size = 52, buffer_count = 1
I1220 02:16:08.130212 539261 hugectr.cc:1665]   requested_output OUTPUT0
I1220 02:16:08.130255 539261 hugectr.cc:1815] *****Processing request on device***** 0 for model dlrm
[HUGECTR][02:16:08][INFO][RANK0]: Looking up 0 embeddings (each with 128 values)...
[HUGECTR][02:16:08][INFO][RANK0]: ParallelLocalMemory backend. Table: dlrm#0. Fetched 0 / 0 values.
[HUGECTR][02:16:08][INFO][RANK0]: ParallelLocalMemory: 0 hits, 0 missing!
[HUGECTR][02:16:08][INFO][RANK0]: Parameter server lookup of 0 / 0 embeddings took 140 us.
I1220 02:16:08.130619 539261 hugectr.cc:1823] ******Processing request completed!*****
I1220 02:16:08.130661 539261 hugectr.cc:1832] Prediction execution time is 0 ms
I1220 02:16:08.130691 539261 hugectr.cc:1438] model dlrm, instance dlrm, executing 1 requests
I1220 02:16:08.130701 539261 hugectr.cc:1512] request 0: id = "", correlation_id = 0, input_count = 3, requested_output_count = 1
I1220 02:16:08.130716 539261 hugectr.cc:1610]   input CATCOLUMN: datatype = INT64, shape = [1,26], byte_size = 208, buffer_count = 1
I1220 02:16:08.130724 539261 hugectr.cc:1626]   input ROWINDEX: datatype = INT32, shape = [1,27], byte_size = 108, buffer_count = 1
I1220 02:16:08.130732 539261 hugectr.cc:1640]   input DES: datatype = FP32, shape = [1,13], byte_size = 52, buffer_count = 1
I1220 02:16:08.130740 539261 hugectr.cc:1665]   requested_output OUTPUT0
I1220 02:16:08.130779 539261 hugectr.cc:1815] *****Processing request on device***** 0 for model dlrm
[HUGECTR][02:16:08][INFO][RANK0]: Looking up 0 embeddings (each with 128 values)...
[HUGECTR][02:16:08][INFO][RANK0]: ParallelLocalMemory backend. Table: dlrm#0. Fetched 0 / 0 values.
[HUGECTR][02:16:08][INFO][RANK0]: ParallelLocalMemory: 0 hits, 0 missing!
I1220 02:16:08.131132 539261 hugectr.cc:1823] ******Processing request completed!*****[HUGECTR][02:16:08][INFO][RANK0]: Parameter server lookup of 0 / 0 embeddings took 151 us.

I1220 02:16:08.131191 539261 hugectr.cc:1832] Prediction execution time is 0 ms
I1220 02:16:08.131238 539261 hugectr.cc:1438] model dlrm, instance dlrm, executing 1 requests
I1220 02:16:08.131250 539261 hugectr.cc:1512] request 0: id = "", correlation_id = 0, input_count = 3, requested_output_count = 1
I1220 02:16:08.131260 539261 hugectr.cc:1610]   input CATCOLUMN: datatype = INT64, shape = [1,26], byte_size = 208, buffer_count = 1
I1220 02:16:08.131269 539261 hugectr.cc:1626]   input ROWINDEX: datatype = INT32, shape = [1,27], byte_size = 108, buffer_count = 1
I1220 02:16:08.131277 539261 hugectr.cc:1640]   input DES: datatype = FP32, shape = [1,13], byte_size = 52, buffer_count = 1
I1220 02:16:08.131287 539261 hugectr.cc:1665]   requested_output OUTPUT0
I1220 02:16:08.131328 539261 hugectr.cc:1815] *****Processing request on device***** 0 for model dlrm
[HUGECTR][02:16:08][INFO][RANK0]: Looking up 0 embeddings (each with 128 values)...
I1220 02:16:08.131681 539261 hugectr.cc:1823] ******Processing request completed!*****
[HUGECTR][02:16:08][INFO][RANK0]: ParallelLocalMemory backend. Table: dlrm#0. Fetched 0 / 0 values.
[HUGECTR][02:16:08][INFO][RANK0]: ParallelLocalMemory: 0 hits, 0 missing!
[HUGECTR][02:16:08][INFO][RANK0]: Parameter server lookup of 0 / 0 embeddings took 170 us.
I1220 02:16:08.131716 539261 hugectr.cc:1832] Prediction execution time is 0 ms
I1220 02:16:08.131747 539261 hugectr.cc:1438] model dlrm, instance dlrm, executing 1 requests
I1220 02:16:08.131757 539261 hugectr.cc:1512] request 0: id = "", correlation_id = 0, input_count = 3, requested_output_count = 1
I1220 02:16:08.131767 539261 hugectr.cc:1610]   input CATCOLUMN: datatype = INT64, shape = [1,26], byte_size = 208, buffer_count = 1
I1220 02:16:08.131773 539261 hugectr.cc:1626]   input ROWINDEX: datatype = INT32, shape = [1,27], byte_size = 108, buffer_count = 1
I1220 02:16:08.131778 539261 hugectr.cc:1640]   input DES: datatype = FP32, shape = [1,13], byte_size = 52, buffer_count = 1
I1220 02:16:08.131786 539261 hugectr.cc:1665]   requested_output OUTPUT0
I1220 02:16:08.131836 539261 hugectr.cc:1815] *****Processing request on device***** 0 for model dlrm
[HUGECTR][02:16:08][INFO][RANK0]: Looking up 0 embeddings (each with 128 values)...
I1220 02:16:08.132182 539261 hugectr.cc:1823] ******Processing request completed!*****
[HUGECTR][02:16:08][INFO][RANK0]: ParallelLocalMemory backend. Table: dlrm#0. Fetched 0 / 0 values.
I1220 02:16:08.132213 539261 hugectr.cc:1832] Prediction execution time is 0 ms
[HUGECTR][02:16:08][INFO][RANK0]: ParallelLocalMemory: 0 hits, 0 missing!
[HUGECTR][02:16:08][INFO][RANK0]: Parameter server lookup of 0 / 0 embeddings took 184 us.
I1220 02:16:08.132245 539261 hugectr.cc:1438] model dlrm, instance dlrm, executing 1 requests
I1220 02:16:08.132256 539261 hugectr.cc:1512] request 0: id = "", correlation_id = 0, input_count = 3, requested_output_count = 1
I1220 02:16:08.132266 539261 hugectr.cc:1610]   input CATCOLUMN: datatype = INT64, shape = [1,26], byte_size = 208, buffer_count = 1
I1220 02:16:08.132273 539261 hugectr.cc:1626]   input ROWINDEX: datatype = INT32, shape = [1,27], byte_size = 108, buffer_count = 1
I1220 02:16:08.132278 539261 hugectr.cc:1640]   input DES: datatype = FP32, shape = [1,13], byte_size = 52, buffer_count = 1
I1220 02:16:08.132282 539261 hugectr.cc:1665]   requested_output OUTPUT0
I1220 02:16:08.132310 539261 hugectr.cc:1815] *****Processing request on device***** 0 for model dlrm
[HUGECTR][02:16:08][INFO][RANK0]: Looking up 0 embeddings (each with 128 values)...
I1220 02:16:08.132655 539261 hugectr.cc:1823] ******Processing request completed!*****
[HUGECTR][02:16:08][INFO][RANK0]: ParallelLocalMemory backend. Table: dlrm#0. Fetched 0 / 0 values.
I1220 02:16:08.132684 539261 hugectr.cc:1832] Prediction execution time is 0 ms[HUGECTR][02:16:08][INFO][RANK0]: ParallelLocalMemory: 0 hits, 0 missing!
[HUGECTR][02:16:08][INFO][RANK0]: Parameter server lookup of 0 / 0 embeddings took 199 us.

I1220 02:16:08.132756 539261 hugectr.cc:1438] model dlrm, instance dlrm, executing 1 requests
I1220 02:16:08.132767 539261 hugectr.cc:1512] request 0: id = "", correlation_id = 0, input_count = 3, requested_output_count = 1
I1220 02:16:08.132779 539261 hugectr.cc:1610]   input CATCOLUMN: datatype = INT64, shape = [1,26], byte_size = 208, buffer_count = 1
I1220 02:16:08.132808 539261 hugectr.cc:1626]   input ROWINDEX: datatype = INT32, shape = [1,27], byte_size = 108, buffer_count = 1
I1220 02:16:08.132817 539261 hugectr.cc:1640]   input DES: datatype = FP32, shape = [1,13], byte_size = 52, buffer_count = 1
I1220 02:16:08.132822 539261 hugectr.cc:1665]   requested_output OUTPUT0
I1220 02:16:08.132859 539261 hugectr.cc:1815] *****Processing request on device***** 0 for model dlrm
[HUGECTR][02:16:08][INFO][RANK0]: Looking up 0 embeddings (each with 128 values)...
[HUGECTR][02:16:08][INFO][RANK0]: ParallelLocalMemory backend. Table: dlrm#0. Fetched 0 / 0 values.
I1220 02:16:08.133222 539261 hugectr.cc:1823] ******Processing request completed!*****[HUGECTR][02:16:08][INFO][RANK0]: ParallelLocalMemory: 0 hits, 0 missing!

[HUGECTR][02:16:08][INFO][RANK0]: Parameter server lookup of 0 / 0 embeddings took 161 us.
I1220 02:16:08.133269 539261 hugectr.cc:1832] Prediction execution time is 0 ms
I1220 02:16:08.133317 539261 hugectr.cc:1438] model dlrm, instance dlrm, executing 1 requests
I1220 02:16:08.133330 539261 hugectr.cc:1512] request 0: id = "", correlation_id = 0, input_count = 3, requested_output_count = 1
I1220 02:16:08.133345 539261 hugectr.cc:1610]   input CATCOLUMN: datatype = INT64, shape = [1,26], byte_size = 208, buffer_count = 1
I1220 02:16:08.133354 539261 hugectr.cc:1626]   input ROWINDEX: datatype = INT32, shape = [1,27], byte_size = 108, buffer_count = 1
I1220 02:16:08.133364 539261 hugectr.cc:1640]   input DES: datatype = FP32, shape = [1,13], byte_size = 52, buffer_count = 1
I1220 02:16:08.133377 539261 hugectr.cc:1665]   requested_output OUTPUT0
I1220 02:16:08.133421 539261 hugectr.cc:1815] *****Processing request on device***** 0 for model dlrm
[HUGECTR][02:16:08][INFO][RANK0]: Looking up 0 embeddings (each with 128 values)...
[HUGECTR][02:16:08][INFO][RANK0]: ParallelLocalMemory backend. Table: dlrm#0. Fetched 0 / 0 values.
[HUGECTR][02:16:08][INFO][RANK0]: ParallelLocalMemory: 0 hits, 0 missing!
I1220 02:16:08.133783 539261 hugectr.cc:1823] ******Processing request completed!*****[HUGECTR][02:16:08][INFO][RANK0]: Parameter server lookup of 0 / 0 embeddings took 150 us.

I1220 02:16:08.133856 539261 hugectr.cc:1832] Prediction execution time is 0 ms
I1220 02:16:08.133894 539261 hugectr.cc:1438] model dlrm, instance dlrm, executing 1 requests
I1220 02:16:08.133903 539261 hugectr.cc:1512] request 0: id = "", correlation_id = 0, input_count = 3, requested_output_count = 1
I1220 02:16:08.133917 539261 hugectr.cc:1610]   input CATCOLUMN: datatype = INT64, shape = [1,26], byte_size = 208, buffer_count = 1
I1220 02:16:08.133925 539261 hugectr.cc:1626]   input ROWINDEX: datatype = INT32, shape = [1,27], byte_size = 108, buffer_count = 1
I1220 02:16:08.133931 539261 hugectr.cc:1640]   input DES: datatype = FP32, shape = [1,13], byte_size = 52, buffer_count = 1
I1220 02:16:08.133937 539261 hugectr.cc:1665]   requested_output OUTPUT0
I1220 02:16:08.133976 539261 hugectr.cc:1815] *****Processing request on device***** 0 for model dlrm
[HUGECTR][02:16:08][INFO][RANK0]: Looking up 0 embeddings (each with 128 values)...
[HUGECTR][02:16:08][INFO][RANK0]: ParallelLocalMemory backend. Table: dlrm#0. Fetched 0 / 0 values.
[HUGECTR][02:16:08][INFO][RANK0]: ParallelLocalMemory: 0 hits, 0 missing!
I1220 02:16:08.134336 539261 hugectr.cc:1823] ******Processing request completed!*****[HUGECTR][02:16:08][INFO][RANK0]: Parameter server lookup of 0 / 0 embeddings took 144 us.

I1220 02:16:08.134390 539261 hugectr.cc:1832] Prediction execution time is 0 ms
I1220 02:16:08.134424 539261 hugectr.cc:1438] model dlrm, instance dlrm, executing 1 requests
I1220 02:16:08.134434 539261 hugectr.cc:1512] request 0: id = "", correlation_id = 0, input_count = 3, requested_output_count = 1
I1220 02:16:08.134444 539261 hugectr.cc:1610]   input CATCOLUMN: datatype = INT64, shape = [1,26], byte_size = 208, buffer_count = 1
I1220 02:16:08.134451 539261 hugectr.cc:1626]   input ROWINDEX: datatype = INT32, shape = [1,27], byte_size = 108, buffer_count = 1
I1220 02:16:08.134455 539261 hugectr.cc:1640]   input DES: datatype = FP32, shape = [1,13], byte_size = 52, buffer_count = 1
I1220 02:16:08.134458 539261 hugectr.cc:1665]   requested_output OUTPUT0
I1220 02:16:08.134488 539261 hugectr.cc:1815] *****Processing request on device***** 0 for model dlrm
terminate called after throwing an instance of 'std::system_error'
  what():  Resource temporarily unavailable
Signal (6) received.
 0# 0x000056510D01E8A9 in tritonserver
 1# 0x00007F263065A210 in /usr/lib/x86_64-linux-gnu/libc.so.6
 2# gsignal in /usr/lib/x86_64-linux-gnu/libc.so.6
 3# abort in /usr/lib/x86_64-linux-gnu/libc.so.6
 4# 0x00007F2630A10911 in /usr/lib/x86_64-linux-gnu/libstdc++.so.6
 5# 0x00007F2630A1C38C in /usr/lib/x86_64-linux-gnu/libstdc++.so.6
 6# 0x00007F2630A1C3F7 in /usr/lib/x86_64-linux-gnu/libstdc++.so.6
 7# 0x00007F2630A1C6A9 in /usr/lib/x86_64-linux-gnu/libstdc++.so.6
 8# std::__throw_system_error(int) in /usr/lib/x86_64-linux-gnu/libstdc++.so.6
 9# 0x00007F2630A490BD in /usr/lib/x86_64-linux-gnu/libstdc++.so.6
10# HugeCTR::embedding_cache<long long>::look_up(void const*, std::vector<unsigned long, std::allocator<unsigned long> > const&, float*, HugeCTR::MemoryBlock*, std::vector<CUstream_st*, std::allocator<CUstream_st*> > const&, float) in /usr/local/hugectr/lib/libhugectr_inference.so
11# HugeCTR::InferenceSession::predict(float*, void*, int*, float*, int) in /usr/local/hugectr/lib/libhugectr_inference.so
12# 0x00007F261805B04A in /usr/local/hugectr/backends/hugectr/libtriton_hugectr.so
13# TRITONBACKEND_ModelInstanceExecute in /usr/local/hugectr/backends/hugectr/libtriton_hugectr.so
14# 0x00007F26311E683A in /opt/tritonserver/bin/../lib/libtritonserver.so
15# 0x00007F26311E704D in /opt/tritonserver/bin/../lib/libtritonserver.so
16# 0x00007F263109B801 in /opt/tritonserver/bin/../lib/libtritonserver.so
17# 0x00007F26311E0DC7 in /opt/tritonserver/bin/../lib/libtritonserver.so
18# 0x00007F2630A48DE4 in /usr/lib/x86_64-linux-gnu/libstdc++.so.6
19# 0x00007F2630EC6609 in /usr/lib/x86_64-linux-gnu/libpthread.so.0
20# clone in /usr/lib/x86_64-linux-gnu/libc.so.6

Aborted (core dumped)
但没有生成core文件

root@13609cdec7a0:/opt/tritonserver# ll
total 423640
drwxr-xr-x 1 root          root               4096 Nov 29 08:07  ./
drwxr-xr-x 1 root          root               4096 Oct 21 18:44  ../
-rw-r--r-- 1 root          root                641 Nov  4 19:20 '=1.21.6'
-rw-rw-r-- 1 triton-server triton-server      1485 Oct 21 17:20  LICENSE
-rw-rw-r-- 1 triton-server triton-server   3012640 Oct 21 17:20  NVIDIA_Deep_Learning_Container_License.pdf
-rw-rw-r-- 1 triton-server triton-server         7 Oct 21 17:20  TRITON_VERSION
drwxr-xr-x 1 triton-server triton-server      4096 Nov  4 19:33  backends/
drwxr-xr-x 2 triton-server triton-server      4096 Oct 21 18:45  bin/
drwxr-xr-x 1 triton-server triton-server      4096 Oct 21 18:45  include/
drwxr-xr-x 2 triton-server triton-server      4096 Oct 21 18:45  lib/
-rw------- 1 root          root          430725428 Dec 16 06:00  nohup.out
-rwxrwxr-x 1 triton-server triton-server      4090 Oct 21 17:20  nvidia_entrypoint.sh*
drwxr-xr-x 3 triton-server triton-server      4096 Oct 21 18:48  repoagents/
drwxr-xr-x 2 triton-server triton-server      4096 Oct 21 18:45  third-party-src/

**Comment by yingcanw** [comment]
Server for some reason returned an empty response. Can you share server side logs with --log-verbose=1? and share more detailed log when launch tritonserver.

**Comment by yingcanw** [comment]
This issue will be fixed  in the Feb 2022 release.

**Comment by yingcanw** [comment]
Fixed in V3.4
